sound.c

```diff
@@ -18,9 +18,9 @@ int music(char **env, char *track)
 	char *player;
 
 	#ifdef __linux__
-		player = "/usr/bin/paplay";
+		player = "/usr/bin/mpg123";
```

Comment:
When launching on linux, the player is printing info into terminal. Maybe include "-q" flag, so that the terminal remains clear for future debugging.